app/parse.py

```diff
@@ -1,4 +1,12 @@
-from dataclasses import dataclass
+import collections
+import csv
+from dataclasses import dataclass, fields
+from urllib.parse import urljoin
+
+import requests
+from bs4 import Tag, BeautifulSoup
+
+BASE_URL = "https://quotes.toscrape.com/"
```

Comment:
don't let BASE_URL like this, place it in a function or program start-point

Reply:
I thought this constant could be placed at the top of the program